extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -106,7 +106,8 @@ export let SizeInfoDef;
 /** @typedef {{
       minifiedCreative: string,
       customElementExtensions: !Array<string>,
-      customStylesheets: !Array<{href: string}>
+      customStylesheets: !Array<{href: string}>,
+      ampImages: Array<string>|undefined,
```

Comment:
gulp check-types didn't like this. I suggest:
```
ampImages: (Array<string>|undefined),
```
See: https://github.com/ampproject/amphtml/blob/8ac8e1906b1fffdd031b2e46a87b22977df48b8a/extensions/amp-animation/0.1/web-animation-types.js#L88